src/renderers/WebGLRenderTarget.js

```diff
@@ -37,6 +37,8 @@ function WebGLRenderTarget( width, height, options ) {
 	this.stencilBuffer = options.stencilBuffer !== undefined ? options.stencilBuffer : true;
 	this.depthTexture = options.depthTexture !== undefined ? options.depthTexture : null;
 
+	this.texture.name = options.name !== undefined ? options.name : "";
```

Comment:
I am setting the name of the internal texture, thus it is copied by the texture.copy() function -- which I did add it to: https://github.com/mrdoob/three.js/pull/10304/files#diff-955123916e8856f9a39f789ef5bb406dR83

Basically I cheaped out by only setting the texture name, but that is fine because there is already a texture associated with a render target, and the relationship is 1-to-1 between renderTarget and texture.